docs/topics_5/migration-paths.adoc

```diff
@@ -20,6 +20,7 @@ The following table describes the most common supported migration paths.
 |OpenJDK 8{nbsp}&{nbsp}11
```

Comment:
OpenShift Container Platform is not rendering well in the header. In the paragraph above the table (MTA provides a comprehensive set of rules to assess the suitability of your applications for containerization and deployment on Red Hat OpenShift Container Platform. You can run an MTA analysis to assess your applications' suitability for migration to multiple target platforms.), change 'Red Hat OpenShift Container Platform' to 'Red Hat OpenShift Container Platform (RHOCP)' and use RHOCP in the table header.